plugins/env/README.md

```diff
@@ -1,4 +1,39 @@
+## Env Plugin
 Plugin to manage ENV settings for projects and write .env files during deploy.
 
 Includes `/projects/:permalink/environment?deploy_group=permalink` endpoint that returns the .env content
 for a project and deploy_group.
+
+## GitHub to manage environment variables
+This plugin has an option to use GitHub repository for .env files.  Using GitHub provides better control
+of the environment variables applied to each deploy group and provides the ability to have peer reveiw of 
+environment variable changes.  Each project must opt-in to use this feature.
+
+To enable the feature for an instance of Samson add the environment variable DEPLOYMENT_ENV_REPO and set it 
+to the the organization and repository that has the .env files.   You should set this in the .env file used
+by the samson rails application.
```

Comment:
`You should set this in the .env file used by the samson rails application` <- varies by deployment, just say "set the samson environment variable `xyz` to blah blah, for example org/repo"

Reply:
I'll make this change

Reply:
I rephrased it completely